pkg/entities/tokens.go

```diff
@@ -44,38 +42,36 @@ func (e *Entity) GetIDAndName(symbol string) (string, string, error) {
 	return "", "", err
 }
 
-func (e *Entity) GetChainIdBySymbol(symbol string) (int, error) {
-	mapChainChainId := map[string]string{
-		"eth":    "1",
-		"heco":   "128",
-		"bsc":    "56",
-		"matic":  "137",
-		"op":     "10",
-		"btt":    "199",
-		"okt":    "66",
-		"movr":   "1285",
-		"celo":   "42220",
-		"metis":  "1088",
-		"cro":    "25",
-		"xdai":   "0x64",
-		"boba":   "288",
-		"ftm":    "250",
-		"avax":   "0xa86a",
-		"arb":    "42161",
-		"aurora": "1313161554",
+func (e *Entity) GetChainIdBySymbol(symbol string) ([]model.Chain, error) {
+	listChain, err := e.repo.Chain.GetAll()
+	var returnChain []model.Chain
+	if err != nil {
+		return listChain, err
```

Comment:
if errors, return []model.Chain. We dont know what listChain is when having error